.gitlab-ci.yml

```diff
@@ -3,7 +3,7 @@ stages:
   - binary_build
   
 variables:
-  DATADOG_AGENT_WINBUILDIMAGES: v3946939-487746f
+  DATADOG_AGENT_WINBUILDIMAGES: v4603787-d533a1c
```

Comment:
Docker image didn't have web targets installed. As I want to re-use the same build steps in both environments, I needed to add them. Created a PR (https://github.com/DataDog/datadog-agent-buildimages/pull/135) to the Windows Build image to add them.